opentracing-util/src/main/java/io/opentracing/util/GlobalTracer.java

```diff
@@ -124,6 +123,11 @@ public static synchronized boolean isRegistered() {
     }
 
     @Override
+    public ScopeManager scopeManager() {
```

Comment:
Wondering whether this is necessary - maybe better to have the `GlobalTracer` just deal with tracers, and once a tracer is retrieved then can access its scope manager?

Reply:
There's not currently a way to get the underlying tracer from `GlobalTracer`, so in effect it must implement all methods.

Reply:
Good point - I was just looking at the change rather than remembering the interface being implemented :)